copilot-chat-request.el

```diff
@@ -92,7 +91,7 @@ If your browser does not open automatically, browse to %s."
                ("editor-plugin-version" . "CopilotChat.nvim/2.0.0")
                ("user-agent" . "CopilotChat.nvim/2.0.0")
                ("editor-version" . "Neovim/0.10.0"))
-    :parser 'json-read
+    :parser (apply-partially 'json-parse-buffer :object-type 'alist :false-object :json-false :null-object nil)
```

Comment:
`json.c` does not use `:json-false`, but `:false` is assigned by default, so there is no need to force a custom setting.